ComposureWatch/src/Components/Card/CreateCard.jsx

```diff
@@ -35,6 +35,14 @@ const CreateCard = () => {
     setCardVar(currentCards);
   };
 
+  const cardRemoveChangeHandler = (enteredId) => {
+    setCards((prevCards) => {
+      return prevCards.filter((object) => {
```

Comment:
I generally don't like naming things object or class or things of that nature. It would probably make more sense to call it `card` or even `c` if you prefer (we could talk about single letter variables ... I am not against them as long as you are not using it too far from where it is initialized).